include/neml2/tensors/SR2.h

```diff
@@ -116,4 +118,14 @@ class SR2 : public FixedDimTensor<SR2, 6>
   /// Transpose, no-op
   SR2 transpose() const;
 };
+
+/// Product w_ik e_kj - e_ik w_kj with e SR2 and w WR2
+SR2 product_wemew(const SR2 & e, const WR2 & w);
```

Comment:
Two questions:
1. Should this be a static method? I don't think so, so probably `w` should be the only argument. `e` is just `this`.
2. Can we come up with a better name? I'd suggest something like `skewify`...

Reply:
This wasn't a static method, but a separate binary operator.  I'll move it to math and think of a better name.